docs/tutorials/learning.rst

```diff
@@ -0,0 +1,151 @@
+Installation
+=============
+
+Official Qiskit Experiments releases can be installed via the python package manager 
+``pip``.
+
+.. code-block:: console
+
+    python -m pip install qiskit-experiments
+
+If you want to install the most up-to-date version instead (may not be stable), you can
+install the latest main branch:
+
+.. code-block:: console
+
+    python -m pip install git+https://github.com/Qiskit/qiskit-experiments.git
+
+If you want to develop the package, you can install Qiskit Experiments from source by 
+cloning the repository:
+
+.. code-block:: console
+
+    git clone https://github.com/Qiskit/qiskit-experiments.git
+    python -m pip install -e qiskit-experiments
+
+The ``-e`` option will keep your installed package up to date as you make or pull new 
+changes.
+
+Running your first experiment
+=============================
+
+Let's run a T1 experiment. First, we have to import the T1 experiment from the 
+Qiskit Experiments library:
+
+.. jupyter-execute::
+
+    from qiskit_experiments.library import T1
+
+Experiments must be run on a backend. We're going to use a simulator, 
+:class:`qiskit.providers.fake_provider.FakeVigo`, for 
+this example, but you can use any IBM backend that you can access through Qiskit.
```

Comment:
Is it OK that we're old-fashioned here, not mentioning Qiskit Runtime and BackendV2 ?

Reply:
I would prefer to link to other documentation that explains the different types of backends and runtime—is there a good resource? If not, maybe writing a how-to guide on backends can be a future PR.

Reply:
Ok, nothing to do here